.github/workflows/demos.yml

```diff
@@ -38,11 +38,16 @@ jobs:
       - name: Install Flutter
         uses: subosito/flutter-action@v2
         with:
-          flutter-version: '3.x'
-          channel: 'stable'
+          flutter-version: "3.x"
+          channel: "stable"
       - name: Install melos
         run: flutter pub global activate melos
       - name: Install dependencies
         run: melos prepare
+      - name: Download powersync binary
+        run: |
+          github="https://github.com/powersync-ja/powersync-sqlite-core/releases/download/v0.1.6"
+
+          curl "${github}/libpowersync_x64.so" -o packages/powersync/libpowersync.so --create-dirs -L -f
```

Comment:
I feel like we should have a Dart Script which downloads the platform specific binary - to make running tests locally easier for developers. 

We currently have scripts for web workers and SQLite WASM which are triggered when running `melos prepare` - I think it should be relatively easy to add a script for this.

Reply:
The only problem is that this needs to be platform specific. So the script will have to detect their platform and download the respective binary.

Reply:
The platform should be available in the Dart script with:

```Dart
import 'dart:io' show Platform;
```
which is the reason for recommending using a Dart script.

Reply:
Done, had to check the Abi specifically because of the differences between x64 and arm.